tests/telemetry/test_telemetry.py

```diff
@@ -0,0 +1,59 @@
+from common.helpers.assertions import pytest_assert
+
+# Helper functions
+def get_dict_stdout(gnmi_out, certs_out):
+    """ Extracts dictionary from redis output.
+    """
+    gnmi_list = []
+    gnmi_list = get_list_stdout(gnmi_out) + get_list_stdout(certs_out)
+    # Elements in list alternate between key and value. Separate them and combine into a dict.
+    key_list = gnmi_list[0::2]
+    value_list = gnmi_list[1::2]
+    params_dict = dict(zip(key_list, value_list))
+    return params_dict
+
+def get_list_stdout(cmd_out):
+    out_list = []
+    for x in cmd_out:
+        result = x.encode('UTF-8')
+        out_list.append(result)
+    return out_list
+
+# Test functions
+def test_config_db_parameters(duthost):
+    """Verifies required telemetry parameters from config_db.
+    """
+    gnmi = duthost.shell('/usr/bin/redis-cli -n 4 hgetall "TELEMETRY|gnmi"', module_ignore_errors=False)['stdout_lines']
+    certs = duthost.shell('/usr/bin/redis-cli -n 4 hgetall "TELEMETRY|certs"', module_ignore_errors=False)['stdout_lines']
+    if gnmi is None or certs is None:
```

Comment:
This `if` statement is redundant here, because the assert statements will catch these cases and exit.